### DISCUSSION TITLE
Jump to timeline from search result.

### DISCUSSION THREAD

**Discussion by denmalley** [immich-app/immich#8986] [General]
Often I'll search up something, maybe by some item - the AI works amazingly well for a lot of different search terms like "ladder" if you're trying to say, find some time you did work on your house.  But then you found a pic from that time, and you have to note the date, back out, jump to timeline and scroll to the time in question.  Be nice to just select something like "go to timeline" to just jump there so you can quickly browse accompanying pics.

**Comment by mertalev** [comment]
It'd definitely be a killer feature. We want to implement this eventually, but it's complicated because of how the timeline gets built.

**Comment by bo0tzz** [comment]
#4115
